pennylane/io/qualtran_io.py

```diff
@@ -0,0 +1,293 @@
+# Copyright 2018-2025 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This submodule contains the adapter class for Qualtran-PennyLane interoperability.
+"""
+import numpy as np
+
+import pennylane as qml
+from pennylane.operation import DecompositionUndefinedError, MatrixUndefinedError, Operation
+from pennylane.wires import WiresLike
+
+try:
+    import qualtran as qt
+except (ModuleNotFoundError, ImportError) as import_error:
+    pass
+
+
+def get_bloq_registers_info(bloq):
+    """Returns a ``qml.registers`` object associated with all named and unnamed registers and wires
+    in the bloq.
+
+    Args:
+        bloq (Bloq): the bloq to get the registers info of
```

Comment:
Update the description to not have "of" at the end. Make sure that the description is clear.

Reply:
Updated. Forgot to push the commit yesterday.